cat/CaT-service.yaml

```diff
@@ -4264,12 +4268,77 @@ components:
                   $ref: '#/components/schemas/QuestionRelationship'
         OCDS:
           $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/components/schemas/Requirement'
+          
+    TableDefinition:
+      type: object
+      properties:
+        editableCols: 
+          type: boolean
+        editableRows: 
+          type: boolean
+        titles:
+          type: object
+          properties:
+            columns:
+              type: array
+              items:
+                allOf:
+                  - $ref: '#/components/schemas/TitleDefinition'
+                  - type: object
```

Comment:
Swagger editor error:
```
Structural error at components.schemas.TableDefinition.properties.titles.properties.columns.items.allOf.1
should NOT have additional properties
additionalProperty: name
Jump to line 4287
```

Reply:
@tommyb82 , I was trying to give a name to the combined allOf object. We can ignore it and clean up in the future. It should not make any difference to the computer.